### ISSUE TITLE
mincLogLikRatioParametricBootstrap doesn't handle vertex mask properly:

### ISSUE THREAD

**Issue by gdevenyi** [Mouse-Imaging-Centre/RMINC#229]
```
mincLogLikRatioParametricBootstrap(loglik1)

Error in minc.dimensions.sizes(filename) :
 Error opening input file: /opt/quarantine/resources/CIVET-CC-mask.txt.
/opt/quarantine/minc-toolkit/1.9.16/src/minc-toolkit-v2/libminc/libsrc/netcdf_convenience.c:573 (from miopen): Unable to open file '/opt/quarantine/resources/CIVET-CC-mask.txt'
/opt/quarantine/resources/CIVET-CC-mask.txt: No such file or directory
/opt/quarantine/minc-toolkit/1.9.16/src/minc-toolkit-v2/libminc/libsrc2/volume.c:1386 (from miopen): Unable to open file '/opt/quarantine/resources/CIVET-CC-mask.txt'
```

**Comment by gdevenyi** [comment]
Ideally, there probably should be a vertexLogLikRatioParametricBootstrap and anatLogLikRatioParametricBootstrap

**Comment by cfhammill** [comment]
I'm hesitant to invest time in this, although should be pretty quick. My sincere hope is that we won't be using that for model comparison in the future, instead using one of the approximate cross-validation techniques. Although I wonder if these bootstrap samples could be used to compute wAIC instead...

**Comment by cfhammill** [comment]
Is that actually what you want? It is easier and more aligned with my aesthetics.

**Comment by gdevenyi** [comment]
Only reason I'm using this is because its the only thing available for *lmer right now.

**Comment by cfhammill** [comment]
Ok, I'll try to do that this week.

**Comment by gdevenyi** [comment]
Closed in favour of #230
